psga/common/primer_autodetection.nf

```diff
@@ -2,19 +2,17 @@
  * Run: primer-autodetection (use first read only for illumina)
  */
 process primer_autodetection {
-  publishDir "${params.output_path}/primer_autodetection", mode: 'copy', overwrite: true, pattern: '{*_primer_data.csv,*_primer_detection.csv,trimmomatic.out,*.bowtie2*}'
```

Comment:
I removed these other output files as the two csv files contain enough information anyway.